repo/fsrepo/fsrepo_test.go

```diff
@@ -30,31 +30,7 @@ func TestInitIdempotence(t *testing.T) {
 func TestRemove(t *testing.T) {
 	t.Parallel()
 	path := testRepoPath("foo", t)
-	assert.Nil(Remove(path), t, "should be able to remove after closed")
-}
-
-func TestCannotRemoveIfOpen(t *testing.T) {
```

Comment:
Well, actually, `ipfs init` doesn't check for other running processes, I remembered that wrong. What it does is test for existence, and normally bail out with an error. With `--force`, it instead removes the repo, and initializes it from scratch. 

That doesn't change my understanding though, and this commit did not change real behavior. The `ipfs init` itself didn't hold it open, so the old refcounting saw nothing special there.